### ISSUE TITLE
An unhandled exception (”undefined is null or not an object”) occured in SLMIV.exe [7144]

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Animeimp/slmiv#17]
```
What steps will reproduce the problem?
1. Sign into SL using SL Connect tab

"looks to be coming from
http://secondlife.com/app/login/_scripts/need_new_version.js - from when
you load up the second life website in “SL Connection” tab. It tries to
split the version b parts.
var version_b_parts = version_b.split(’.');"

WORKAROUND: Disconnect your internet connection, then run SLMIV, load the
SL Connect tab and the website can not be displayed, then connect back to
the internet. You should be able to sign in after that, (with large
inventories it will take about 3 minutes to create the SLMIV tree).

(submitted by Dedric Mauriac using Microsoft Vista 64 bit)


```

Original issue reported on code.google.com by `joesox` on 26 Jan 2009 at 2:25

**Comment by GoogleCodeExporter** [comment]
```
Fixed/Workaround in version 2.10.17. The webpage is not displayed by default.
```

Original comment by `joesox` on 9 Feb 2009 at 3:01

**Comment by GoogleCodeExporter** [comment]
```
closing
```

Original comment by `joesox` on 9 Feb 2009 at 3:01
- Changed state: **Fixed**
